mage_integrations/requirements.txt

```diff
@@ -21,10 +21,12 @@ redshift-connector==2.0.909
 requests==2.27.1
 requests_mock==1.10.0
 simplejson==3.11.1
+singer-python==5.13.0
```

Comment:
Needed to add this to resolve a `singer` module import error.

Reply:
We use a forked singer-python package: https://github.com/mage-ai/singer-python
You can install it via `pip install "git+https://github.com/mage-ai/singer-python.git#egg=singer-python"`